roles/2-common/tasks/debian.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
These days it's prob better to delete the `---` line at the top of most all of our Ansible files?

Reply:
Beginning a file with '---' is recommended YAML syntax. 

`There’s another small quirk to YAML. All YAML files (regardless of their association with Ansible or not) can optionally begin with --- and end with .... This is part of the YAML format and indicates the start and end of a document.` 

https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-syntax